css-ecommerce-page/style.css

```diff
@@ -0,0 +1,50 @@
+* {
+  font-family: DM Sans, sans-serif, Arial;
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+}
+
+h3 {
+  margin-bottom: 0;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+h1 {
+  font-weight: 600;
+}
+
+span {
+  opacity: 0.3;
+}
```

Comment:
Is this to make the text lighter? This is actually making your content more see-through, this is not the ideal way to do this. You should just change the text color to something lighter